src/components/policy/src/policy/src/cache_manager.cc

```diff
@@ -790,8 +790,22 @@ utils::SharedPtr<policy_table::Table>
 CacheManager::GenerateSnapshot() {
   CACHE_MANAGER_CHECK(snapshot_);
   sync_primitives::AutoLock lock(cache_lock_);
+
   snapshot_ = new policy_table::Table();
-  snapshot_->policy_table = pt_->policy_table;
```

Comment:
Because that would copy the entire contents of pt_, and pt_ contains all of the messages in consumer friendly messages. The policy table "snapshot" is no supposed to contain these messages.